integration_tests/src/main/python/benchmark.py

```diff
@@ -113,7 +119,7 @@ def main():
             cmd.append("--input-format {}".format(args.input_format))
 
             if args.append_dat is True:
-                cmd.append("--append-dat ")
```

Comment:
There is no need to add a whitespace after each argument since we are building a list which we later turn into a whitespace-separated string